src/hammer-vlsi/hammer_vlsi/cli_driver.py

```diff
@@ -856,6 +859,77 @@ def auto_action(driver: HammerDriver, append_error_func: Callable[[str], None])
 
         return driver, errors
 
+    @staticmethod
+    def make_build_inputs(driver: HammerDriver, append_error_func: Callable[[str], None]) -> Optional[dict]:
+        """
+        TODO
+        """
+        build_system_key = "vlsi.core.build_system"
```

Comment:
Just curious as to how come this is its own variable

Reply:
Just easier to read- there are other methods that use this pattern, and I modeled after them. I can change it; it's not _that_ much shorter.